examples/tensorflow2/tensorflow2_keras_mnist.py

```diff
@@ -59,7 +54,7 @@ def main():
 
     # Horovod: adjust learning rate based on number of GPUs.
     scaled_lr = 0.001 * hvd.size()
-    opt = optimizers.Adam(scaled_lr)
+    opt = tf.optimizers.Adam(scaled_lr)
```

Comment:
Not related to this PR, but a question about `tf.keras` and `keras` compatibility in horovod:
- After tf 2.6, tf.keras == keras.
- Before tf 2.6, this might be an issue?
- Or our CI already dropped support for TF2 versions < 2.6?

Reply:
I don't know exactly about the test matrix of our CI.
Technically it should work for tf < 2.6 version when using keras package. I maintained the code path for pure keras (older version of TF and Keras).